verifier/src/quote_enclave_identity.rs

```diff
@@ -107,6 +111,46 @@ pub struct SignedQeIdentity<'a> {
     signature: Vec<u8>,
 }
 
+impl<'a> SignedQeIdentity<'a> {
```

Comment:
This is very similar to the [TcbInfoRaw](https://github.com/mobilecoinfoundation/attestation/blob/8413e9c404f158241cd854cc381e46586f48ed30/verifier/src/tcb.rs#L234) implementation.
I'm on the fence on how much to worry about the duplication.

One thought I had was:
```rust
#[derive(Debug, Deserialize, Clone)]
pub struct SignedJson<'a> {
    #[serde(borrow, alias="tcbInfo", alias="enclaveIdentity")]
    primary_field: &'a RawValue,
    #[serde(with = "hex")]
    signature: Vec<u8>,
}

#[derive(Debug, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct SignedJsonTime {
    issue_date: String,
    next_update: String,
}
```

Reply:
I personally wouldn't worry about the duplication, it's a very small amount of simple to follow code (assuming this only repeats for these two types, if we had a bunch more then maybe I would lean towards DRYing).